packages/@react-spectrum/autocomplete/test/SearchAutocomplete.test.js

```diff
@@ -816,6 +816,43 @@ describe('SearchAutocomplete', function () {
       expect(items).toHaveLength(1);
       expect(searchAutocomplete).not.toHaveAttribute('aria-activedescendant');
     });
+
+    it('should clear the selected item if searchfield is cleared', function () {
+      let {getByRole} = renderSearchAutocomplete();
+
+      let searchAutocomplete = getByRole('combobox');
+      typeText(searchAutocomplete, 'one');
+
+      act(() => {
+        jest.runAllTimers();
+      });
+
+      let listbox = getByRole('listbox');
+      let items = within(listbox).getAllByRole('option');
+      expect(items).toHaveLength(1);
+      expect(searchAutocomplete).not.toHaveAttribute('aria-activedescendant');
+
+      act(() => {
+        fireEvent.keyDown(searchAutocomplete, {key: 'ArrowDown'});
+        fireEvent.keyUp(searchAutocomplete, {key: 'ArrowDown'});
+        fireEvent.keyDown(searchAutocomplete, {key: 'Enter'});
+        fireEvent.keyUp(searchAutocomplete, {key: 'Enter'});
+      });
```

Comment:
do we need this extra act around the fireEvents?
same question for below with esc

Reply:
I suppose not, just copied from a test above. Fixed.

Reply:
ah, yeah, we still have some of those floating around